package.json

```diff
@@ -14,7 +14,7 @@
     "@tanstack/react-query": "^5.50.1",
     "clsx": "^2.1.1",
     "firebase": "^10.12.4",
-    "fuels": "^0.94.9",
+    "fuels": "^0.95.0",
     "graphql": "^16.9.0",
     "graphql-request": "^7.1.0",
     "mira-dex-ts": "^1.0.15",
```

Comment:
```suggestion
    "mira-dex-ts": "^1.0.21",
```

Reply:
Also update `@fuels/connectors` and `@fuels/react` to `0.32.0`.